src/foam/util/Emails/EmailsUtility.java

```diff
@@ -65,6 +65,7 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
 
     // SERVICE CALL: passing emailMessage through to actual email service.
     DAO email = (DAO) x.get("emailMessageDAO");
+    emailMessage.setStatus(foam.nanos.notification.email.Status.UNSENT);
```

Comment:
@mcarcaso  Why do we need an EmailsUtility? Assuming existed for the EmailService. 
If the default is UNSENT, then a simple DAO.put() will take of things.

Reply:
to be removed in future PR